.gitignore

```diff
@@ -28,3 +28,11 @@ docs/html
 
 # Do not track results
 regression_tests/results
+init_bash.sh
+.gitignore
+.vscode/settings.json
+.vscode/libROM.code-workspace
+init_bash_debug_arm.sh
```

Comment:
Are the 2 .sh files unique to your testing? I'm not sure that libROM produces this, and we should only put files here that are commonly produced by libROM. Similarly, I'm not sure we need to include `.vscode` files here. I don't think we need `.gitignore` in `.gitignore` (it should be automatically ignored, but I think that depends on your git settings).

It is common that .txt files are used or generated, and they are usually not part of the repo. Ignoring .txt files would be convenient in my opinion, but I wonder if it we might ever have a txt file in the repo. I suppose if a .txt file is ever added, we will notice it is ignored and can revert this.

Reply:
Right, this is the result of me accidentally commiting my local gitignore file!